presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -311,6 +311,14 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kRemoteFunctionServerCatalogName{
       "remote-function-server.catalog-name"};
 
+  /// Options to configure the internal (in-cluster) JWT authentication.
+  static constexpr std::string_view kInternalCommunicationJwtEnabled{
+      "internal-communication.jwt.enabled"};
+  static constexpr std::string_view kInternalCommunicationSharedSecret{
+      "internal-communication.shared-secret"};
+  static constexpr std::string_view kInternalCommunicationJwtExpirationSeconds{
```

Comment:
Yes, this is new. It wasn't added to Java but it allows better testing and more flexibility. On the Java side it is hard coded to 5 minutes. 
I suppose this comes back to the discussion we had recently on the configuration topic, which config applies to where. This would be a prestissimo only configuration. Though it could be added in Java subsequently.

I added it to the doc. Lets see.